x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/cti/threat_match_rows.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { get } from 'lodash';
+import React from 'react';
+
+import { Ecs } from '../../../../../../../common/ecs';
+import { ParsedFields, TimelineNonEcsData } from '../../../../../../../common/search_strategy';
+import { RowRendererContainer } from '../row_renderer';
+import { ThreatMatchRow } from './threat_match_row';
+
+export const ThreatMatchRows = ({
+  data,
+  flattenedData,
+}: {
+  data: Ecs;
+  flattenedData?: TimelineNonEcsData[];
+}) => {
+  const indicators = get(data, 'threat.indicator') as ParsedFields[];
+
+  return (
+    <RowRendererContainer data-test-subj="threat-match-row-renderer">
+      {indicators.map((fields, index) => (
+        <ThreatMatchRow key={index} fields={fields} />
```

Comment:
There is, but it looks like we're not using it 😅 . We have to use `index` here for now, since `matched.id` isn't mapped and available to the row renderer. I've added a TODO for this, but since the nested field order seems to be consistent we should be okay for now.

Reply:
I added a comment here about this but then you told me to delete it 😢